src/order/category/NonemptyFinLinOrd.lean

```diff
@@ -91,3 +91,8 @@ end NonemptyFinLinOrd
 lemma NonemptyFinLinOrd_dual_comp_forget_to_LinearOrder :
   NonemptyFinLinOrd.dual ⋙ forget₂ NonemptyFinLinOrd LinearOrder =
     forget₂ NonemptyFinLinOrd LinearOrder ⋙ LinearOrder.dual := rfl
+
+/-- The obvious functor `NonemptyFinLinOrd ⥤ Preorder` -/
+def NonemptyFinLinOrd_forget_to_Preorder : NonemptyFinLinOrd ⥤ Preorder :=
```

Comment:
Could you add the full and faithful instances here, and wherever they are relevant above?

Reply:
This functor actually already exists, as `forget₂ NonemptyFinLinOrd LinearOrder⋙ forget₂ LinearOrder Lattice ⋙ forget₂ Lattice PartialOrder ⋙ forget₂ PartiaOrder Preorder`. I don't really see the point of naming all forgetful, as opposed to just the immediate ones.

Reply:
I have fullness and faithfulness of all those on a branch. Should probably get back to it.

Reply:
Thanks for your reviews. Actually, I had noticed that this functor existed as this composition of four functors (even though they are not definitionally equal on maps), but I was not sure what was the preferred option.